src/collegetown.py

```diff
@@ -21,14 +22,23 @@ def collegetown_search():
         longitude=YELP_LONGITUDE,
         radius=YELP_RADIUS
     )
-    eateries = []
-    for place in search['businesses']:
-      eatery_id = place['id']
-      id_query = yelp_api.business_query(id=eatery_id)
-      eateries.append(id_query)
-      sleep(YELP_QUERY_DELAY)  # delay between query calls to not trigger yelp query limits
-    print('Done with Collegetown Search')
-    return eateries
   except Exception as e:
+    print('failed at yelp radial search')
     print(e)
     return []
+
+  eateries = []
+  for place in search.get('businesses', []):
+    eatery_id = place['id']
+    if eatery_id == SWEETSPOT_ID:
+      continue
+    try:
+      id_query = yelp_api.business_query(id=eatery_id)
+    except Exception as e:
+      print('failed at business query with id {}'.format(eatery_id))
+      print(e)
+      continue
+    eateries.append(id_query)
+    sleep(YELP_QUERY_DELAY)  # delay between query calls to not trigger yelp query limits
```

Comment:
we should not `sleep` in code, because that shuts down the entire process and makes it unresponsive, this should be in a thread instead!!!